.changeset/chilly-books-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-neoboard-widget': minor
+---
+
+Images can be uploaded to the board (needs REACT_APP_IMAGES=true)
```

Comment:
```suggestion
Upload images
```
No strong opinion about `REACT_APP_IMAGES=true` part, at least last time with multi select it wasn't mentioned in changelog, in configuration only.

Reply:
As far as I can remember we have discussed that and said that we should mention it. Otherwise from the changelog you may expect a feature that is not there if you just run the board. I will take this to our discussion again.